src/esp/bindings_js/navigate.js

```diff
@@ -12,26 +12,30 @@ class NavigateTask {
   constructor(sim, components) {
     this.sim = sim;
     this.components = components;
-    this.imageCtx = components.canvas.getContext("2d");
-    let shape = this.sim.getObservationSpace("rgb").shape;
+    this.imageCtx = components.canvas.getContext('2d');
+    let shape = this.sim.getObservationSpace('rgb').shape;
     this.imageData = this.imageCtx.createImageData(shape.get(1), shape.get(0));
-    this.semanticCtx = components.semantic.getContext("2d");
-    shape = this.sim.getObservationSpace("semantic").shape;
-    this.semanticImageData = this.semanticCtx.createImageData(shape.get(1), shape.get(0));
+    this.semanticCtx = components.semantic.getContext('2d');
+    shape = this.sim.getObservationSpace('semantic').shape;
+    this.semanticImageData =
```

Comment:
Same here. Maybe we should move to black. Worth checking if there are options there.
Black would make this
```js
this.semanticImageData = this.semanticCtx.createImageData(
    shape.get(1), shape.get(0)
)
```
which looks much nicer.

Reply:
Does black work on javascript?  I thought it was python only

Reply:
Oh yeah. Sorry, working too much with python. There is a way to make it work with JavaScript but it is not worth it. In my mind, I was actually talking about eslint.

Reply:
ESLint is very nice in doing all this and very customizable.

Reply:
Does black support javascript?

I wasn't able to find any option to fix this.